packages/storefront/src/pages/developing/angular/testing.md

```diff
@@ -0,0 +1,135 @@
+# Angular
+
+<TableOfContents></TableOfContents>
+
+# Test the application
+
+**Jest** uses **jsdom** and supports ShadowDOM since Version 12.2.0.  
+However, it doesn't support JavaScript modules as described in this
+[issue](https://github.com/jsdom/jsdom/issues/2475).  
+Also, it doesn't support `CSSStyleSheet.replace()`, `Intersection Observer`, `Element.prototype.scrollTo` and others.
+
+As a workaround we provide a polyfill as part of the `@porsche-design-system/components-angular` package.
+
+To apply the polyfill, simply import it in your **setupTest.{js|ts}** file.
+
+**Note:** If your test includes Porsche Design System components, make sure to import the PorscheDesignSystemModule in
```

Comment:
Maybe better:
"_make sure that you import the PorscheDesignSystemModule into
each module in which you want to use the components._"